<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main_Window</class>
 <widget class="QMainWindow" name="Main_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1176</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Predictive Maintenance</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Icon.png</normaloff>Icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>360</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>338</width>
            <height>670</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QToolBox" name="toolBox">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="page">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>320</width>
                <height>559</height>
               </rect>
              </property>
              <attribute name="label">
               <string>Selection des Données</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QFrame" name="frame_3">
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QFrame" name="frame_7">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>50</height>
                     </size>
                    </property>
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QFormLayout" name="formLayout">
                     <item row="0" column="0">
                      <widget class="QLabel" name="fileName">
                       <property name="minimumSize">
                        <size>
                         <width>150</width>
                         <height>0</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>nom du fichier</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QPushButton" name="browseButton">
                       <property name="maximumSize">
                        <size>
                         <width>100</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Rechercher</string>
                       </property>
                       <property name="icon">
                        <iconset theme="folder">
                         <normaloff>.</normaloff>.</iconset>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame_6">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <widget class="QLabel" name="label_2">
                     <property name="geometry">
                      <rect>
                       <x>10</x>
                       <y>10</y>
                       <width>261</width>
                       <height>17</height>
                      </rect>
                     </property>
                     <property name="text">
                      <string>Information sur le fichier :</string>
                     </property>
                    </widget>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox">
                    <property name="currentText">
                     <string/>
                    </property>
                    <property name="placeholderText">
                     <string>Flotte</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox_2">
                    <property name="placeholderText">
                     <string>Equippement</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox_4">
                    <property name="currentText">
                     <string/>
                    </property>
                    <property name="placeholderText">
                     <string>Numéro de la pièce</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox_5"/>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_3">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>320</width>
                <height>559</height>
               </rect>
              </property>
              <attribute name="label">
               <string>Propriétés du Graphique</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QFrame" name="frame_4">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <widget class="QPushButton" name="pushButton_2">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>20</y>
                    <width>241</width>
                    <height>25</height>
                   </rect>
                  </property>
                  <property name="text">
                   <string>Actualiser</string>
                  </property>
                  <property name="icon">
                   <iconset theme="view-refresh">
                    <normaloff>.</normaloff>.</iconset>
                  </property>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="page_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>320</width>
                <height>559</height>
               </rect>
              </property>
              <attribute name="label">
               <string>Programme d'entretien</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QFrame" name="frame_5">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_4">
                  <item>
                   <widget class="QFrame" name="frame_9">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_5">
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="maximumSize">
                        <size>
                         <width>16777215</width>
                         <height>50</height>
                        </size>
                       </property>
                       <property name="text">
                        <string>Programme d'entretien actuelle :</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignCenter</set>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Numéro de la tâche :&lt;/p&gt;&lt;p&gt;P/N : &lt;/p&gt;&lt;p&gt;Type de suivi :&lt;/p&gt;&lt;p&gt;Périodicité :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton_3">
                    <property name="text">
                     <string>Nouveau programme d'entretien</string>
                    </property>
                    <property name="icon">
                     <iconset theme="appointment-new">
                      <normaloff>.</normaloff>.</iconset>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Total Occurency</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QWidget" name="plotWidget_hist" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Occurency Leading To Part Remplacement</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QWidget" name="plotWidget_hist2" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Linear Regression</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QWidget" name="plotWidget" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1176</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
